### PR TITLE
Update the contracts up to 1.2.5

### PR DESCRIPTION
**Pull request by oleg1std**

### CODE REVIEW

**Comment by oleg1std** [comment]
Do not merge!

File: Api/Models/Accommodations/AvailabilityResult.cs [hunk 1]
```diff
@@ -30,6 +30,6 @@ public AvailabilityResult(AvailabilityResult result, List<Agreement> agreements)
         /// <summary>
         /// List of available agreements
         /// </summary>
-        public List<Agreement> Agreements { get; }
+        public List<RoomContractSet> Agreements { get; }
```
Comment (hasmikharutyunyan) [inline Api/Models/Accommodations/AvailabilityResult.cs:33]: Why not RoomContractSets?

Comment (oleg1std) [reply Api/Models/Accommodations/AvailabilityResult.cs:33]: It's an external model and changing property name can break something on the web-app

Comment (kirillta) [reply Api/Models/Accommodations/AvailabilityResult.cs:33]: That will be done sooner or later. Think it's safe to rename now

File: Api/Services/Connectors/DataProvider.cs [hunk 2]
```diff
@@ -40,21 +40,21 @@ public DataProvider(IDataProviderClient dataProviderClient, string baseUrl, ILog
...
         }
 
 
-        public Task<Result<DeadlineDetails, ProblemDetails>> GetDeadline(string availabilityId, Guid agreementId, string languageCode)
+        public Task<Result<DeadlineDetails, ProblemDetails>> GetDeadline(string availabilityId, Guid roomContractSetId, string languageCode)
         {
             return ExecuteWithLogging(() =>
             {
-                var uri = new Uri($"{_baseUrl}accommodations/availabilities/{availabilityId}/agreements/{agreementId}/deadline", UriKind.Absolute);
+                var uri = new Uri($"{_baseUrl}accommodations/availabilities/{availabilityId}/room-contract-sets/{roomContractSetId}/deadline", UriKind.Absolute);
```
Comment (kirillta) [inline Api/Services/Connectors/DataProvider.cs:57]: Don't forget to change routes in Insomnia

**Pull request merged**